packages/build-lz-options/src/components/outputOptions.tsx

```diff
@@ -15,8 +15,14 @@ export const OutputOptions: React.FC<OptionOutputProps> = ({
 }: OptionOutputProps) => {
   return (
     <Box flexDirection="column">
+      {props.warning.length > 0 && (
+        <Text color={"yellow"}>
+          <Text bold={true}>Warning:</Text> {props.warning}
+        </Text>
+      )}
```

Comment:
I think here `props.warning` is an array of strings so something like this would make more sense:

```jsx
<Box flexDirection="column">
  {props.warning.map(warning => (
    <Text color="yellow">
      <Text bold>Warning:</Text> {warning}
    </Text>
  )}
</Box>
```